src/test/meson.build

```diff
@@ -296,8 +296,7 @@ tests += [
 
         [['src/test/test-selinux.c']],
 
-        [['src/test/test-sizeof.c'],
-         [libbasic]],
+        [['src/test/test-sizeof.c']],
```

Comment:
The default is libshared, and this effectively replaces the libbasic linkage with libshared. Those tests are linked without libshared on purpose, to make it possible to build them very quickly when bootstrapping a build (on some strange architecture or with a new compiler, when building all of libshared is too much).

Reply:
OK. The change is dropped.